app/views/evidence/_breadcrumbs.html.erb

```diff
@@ -1,6 +1,5 @@
 <ol class="breadcrumb">
   <li class="breadcrumb-item"><a href="javascript:void(0)" data-behavior="sidebar-toggle">Nodes</a></li>
-  <li class="breadcrumb-item" data-node-id="<%= @node.id %>"><%= link_to @node.label, main_app.project_node_path(@node.project, @node) %></li>
-  <li class="breadcrumb-item"><%= link_to 'Evidence', main_app.project_node_path(@node.project, @node, tab: 'evidence-tab') %></li>
+  <li class="breadcrumb-item" data-node-id="<%= @node.id %>"><%= link_to @node.label, main_app.project_node_path(@node.project, @node, tab: 'evidence-tab') %></li>
```

Comment:
This PR does more than what it says in the tin, breadcrumb changes, including adding breadcrumbs to revisions is a different concern, split into a separate PR.